dev/ts/shared/main.ts

```diff
@@ -80,7 +80,7 @@ function loadBpmn(bpmn: string, handleError = true): void {
     collapseBpmnElement(bpmnElementIdToCollapse);
     document.dispatchEvent(new CustomEvent('diagramLoaded'));
   } catch (error) {
-    if (handleError) {
+    if (handleError && error instanceof Error) {
```

Comment:
**nitpick:** the error must be sent to the notifier in any case.
Same proposal as in the comment in _dev/ts/component/DropFileUserInterface.ts_